### PR TITLE
Added animefreak.tv

### PR DESCRIPTION
**Pull request by ngomile**
Successfully added animefreak.tv, site seems to only serve videos in one quality since I could not see more qualities.

### CODE REVIEW

File: anime_downloader/sites/animefreak.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+class AnimeFreak(Anime, sitename='animefreak'):
+        sitename = 'animefreak'
+        search_url = f'https://{sitename}.tv/search/topSearch'
+        anime_url = 'https://www.animefreak.tv/watch'
+        QUALITIES = ['360p', '480p', '720p', '1080p']
+
+        @classmethod
+        def search(cls, query):
+            search_results = util.get_json(cls.search_url, {'q': query})
```
Comment (vn-ki) [inline anime_downloader/sites/animefreak.py]: Use `helpers.get().json()` instead

Comment (ngomile) [reply anime_downloader/sites/animefreak.py]: Done.

**Comment by ngomile** [comment]
All necessary changes and additions have been made.

**Comment by vn-ki** [comment]
@ngomile You don't seem to have the added the title to the anime. This means it won't download the titles properly. Can you add it?

Also I have added the tests, so please merge with master before making the change and edit the test too.

**Pull request merged**